uint32 num_msg          # Length of 'table' array
HmsInfoMsg[] table      # Array of HmsInfoMsg

